### ISSUE TITLE
Support npm config params

### ISSUE THREAD

**Issue by vitalets** [mysticatea/npm-run-all#60]
In our scripts we are using `$npm_config_xxx` params.
for example:
```json
"scripts": {
    "zip": "cd $npm_config_outdir && zip -rT $npm_config_outzip .",
    "crx": "crx pack $npm_config_outdir",
    "zip-crx": "npm run zip && npm run crx",
}
```
We can run such scripts via:
```
npm run zip-crx --outdir=dist --outzip=archive.zip
```

it would be great if `npm-run-all` support such npm params and not confuse it with own params or command own params.
Currently I get error:
```
ERROR: Invalid Option: --outdir=dist
```

**Comment by mysticatea** [comment]
Thank you for this issue.

I have not been aware of that feature of npm. (though `npm-run-all` supports `$npm_package_config_xxx` variables since #13).
I will do.

**Comment by vitalets** [comment]
Thanks a lot for quick reply and actions! 
When you plan to release?

**Comment by mysticatea** [comment]
Just now, I released `v3.1.0` :smile: 
Please tell me if there are problems.

Thank you!

**Comment by airtonix** [comment]
This needs to be reverted. it was poorly planned and thought out.
